### DISCUSSION TITLE
PTS Icon

### DISCUSSION THREAD

**Discussion by StevenHue** [guillaumejchauveau/pictoshare#54] [General]
Icon : 
https://www.flaticon.com/free-icon/file_1091609?related_id=1091609&origin=search
https://www.flaticon.com/free-icon/transfering_748615

**Comment by guillaumejchauveau** [comment]
Icon assets and PSD using official MacOS App [PSD template](https://developer.apple.com/design/resources/#macos-apps).
[PTS_Icon.zip](https://github.com/guillaumejchauveau/pictoshare/files/6446635/PTS_Icon.zip)

**Comment by guillaumejchauveau** [comment]
Updated version: [PTS_Icon.zip](https://github.com/guillaumejchauveau/pictoshare/files/6446672/PTS_Icon.zip)
